e3nn/o3/_linear.py

```diff
@@ -489,6 +489,6 @@ def _codegen_linear(
                 bias_numel,
             ),
         )
-        graphmod_out = jitable(optimize_einsums_full(graphmod_out, example_inputs))
+        graphmod_out = optimize_einsums_full(graphmod_out, example_inputs)
```

Comment:
why remove `jitable`? At least, you still need to call it if `jit_script_fx` is true.

I know they un-broke some stuff with the interface compatability between python and torchscript, but is it all fixed? do we know that? and by what version? since we still support 1.8

Reply:
nevermind I see it at the bottom